### ISSUE TITLE
Feature: output the each-time searched parameters not only the best parameters/models

### ISSUE THREAD

**Issue by whubaichuan** [keras-team/autokeras#1838]
### Code Example

``
AutoModel.fit(
    x=None,
    y=None,
    batch_size=32,
    epochs=None,
    callbacks=None,
    validation_split=0.2,
    validation_data=None,
    verbose=1,
    **kwargs
)

Returns

history: A Keras History object corresponding to the best model. Its History.history attribute is a record of training loss values and metrics values at successive epochs, as well as validation loss values and validation metrics values (if applicable).

```

### Reason
<!---
Because I want to add the complexity/latency in the customized metrics to select the best model considering the accuracy and latency at the same time.
-->

### Solution
<!---
No idea
-->
